File: 1021/1021.py
n = float(input())

print('NOTAS:')
print('{} nota(s) de R$ 100.00'.format(int(n // 100)))
n = n%100

print('{} nota(s) de R$ 50.00'.format(int(n // 50)))
n = n%50

print('{} nota(s) de R$ 20.00'.format(int(n // 20)))
n = n%20

print('{} nota(s) de R$ 10.00'.format(int(n // 10)))
n = n%10

print('{} nota(s) de R$ 5.00'.format(int(n // 5)))
n = n%5

print('{} nota(s) de R$ 2.00'.format(int(n // 2)))
n = n%2

print('MOEDAS:')
print('{} moeda(s) de R$ 1.00'.format(int(n // 1)))
n = n%1

print('{} moeda(s) de R$ 0.50'.format(int(float('%.2f'% (n // 0.5)))))
n = n%0.5

print('{} moeda(s) de R$ 0.25'.format(int(float('%.2f'% (n // 0.25)))))
n = n%0.25

print('{} moeda(s) de R$ 0.10'.format(int(float('%.2f'% (n // 0.10)))))
n = n%0.10

print('{} moeda(s) de R$ 0.05'.format(int(float('%.2f'% (n // 0.05)))))
n = n%0.05

print('{} moeda(s) de R$ 0.01'.format(int(float('%.2f'% (n * 100)))))

File: 1. Beginner/1021/1021.js
var lines = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');

let d = parseFloat(lines.shift());
let n = parseInt(d);
let m = parseInt((d - n) * 100);

console.log("NOTAS:");
console.log(parseInt(n / 100) + " nota(s) de R$ 100.00");
n -= parseInt((n / 100)) * 100;

console.log(parseInt(n / 50) + " nota(s) de R$ 50.00");
n -= parseInt((n / 50)) * 50;

console.log(parseInt(n / 20) + " nota(s) de R$ 20.00");
n -= parseInt((n / 20)) * 20;

console.log(parseInt(n / 10) + " nota(s) de R$ 10.00");
n -= parseInt((n / 10)) * 10;

console.log(parseInt(n / 5) + " nota(s) de R$ 5.00");
n -= parseInt((n / 5)) * 5;

console.log(parseInt(n / 2) + " nota(s) de R$ 2.00");
n -= parseInt((n / 2)) * 2;

console.log("MOEDAS:");
console.log(parseInt(n / 1) + " moeda(s) de R$ 1.00");
n -= parseInt(n / 1) * 1;

console.log(parseInt(m / 50) + " moeda(s) de R$ 0.50");
m -= parseInt((m / 50)) * 50;

console.log(parseInt(m / 25) + " moeda(s) de R$ 0.25");
m -= parseInt((m / 25)) * 25;

console.log(parseInt(m / 10) + " moeda(s) de R$ 0.10");
m -= parseInt((m / 10)) * 10;

console.log(parseInt(m / 5) + " moeda(s) de R$ 0.05");
m -= parseInt((m / 5)) * 5;

console.log(m + " moeda(s) de R$ 0.01");